pages/index.js

```diff
@@ -41,15 +42,18 @@ function HomePage() {
 
   return (
     <>
+      <Helmet>
+        <html lang="en" />
```

Comment:
You can put this line here https://github.com/asyncapi/website/blob/b24adac6c75432855c76e9490e0f06babb20a70a/pages/_app.js#L30 to apply for all pages :)

Reply:
> One suggestion, rest looks awesome :)

Thanks, will do that now.

Reply:
> > One suggestion, rest looks awesome :)
> 
> Thanks, will do that now.

I've made the update.

Reply:
`Resolves #692 `

Reply:
You should remove unnecessary code, don't comment 😄